.github/workflows/ci.yml

```diff
@@ -345,6 +345,12 @@ jobs:
         run: rush build -o calling
       - name: Tests
         run: rush test -o calling
+      - name: Upload current calling stats.json
+        uses: actions/upload-artifact@v1
+        if: ${{ matrix.flavor == 'beta' && github.event_name == 'pull_request' }}
```

Comment:
Why only on beta? Might as well upload for both flavors with different file names.

It'd actually be great to see that diff between stable/beta as our calling dependency might be different.

Reply:
This is a little heavy cause it needs to build all previous code so we are adding 3 more apps to build if we build stable - but I agree it would be nice to see the size of stable, let's start with beta, and if everything works great, then I can open a new PR for stable (need to work on some matrix code)